config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
```

Comment:
Be careful on how you order the routes as it does matter. You may want to use the resource helper instead to cover all your standard actions

Reply:
I will refactor with resources keyword later, as I want now to walk through the routes manually. The order is important because when I have a dynamic path identifier it must come after the static address